websites/api-docs-smoke-test/src/content/layouts/type-large-desktop.mdx

```diff
@@ -2,14 +2,43 @@
 title: API Type Large Desktop Layout
 description: Test components that can flow into two content columns
 wideLayout: true
+navLevels: 2
 ---
 
 <Subtitle>
 
-Tests the responsive behavior in "largeDesktop" responsive sizes. It has the `widelayout` frontmatter set to `true`, but would be wide nevertheless because the whole site is wide.
+Tests the responsive behavior in "largeDesktop" responsive sizes. It has the `widelayout` frontmatter set to `true`, but would be wide nevertheless because the whole site is wide. It contains a child section navigation as well.
 
 </Subtitle>
 
+# Representations
+
+<ChildSectionsNav parent="representations"/>
+
+### Darth Vader
+
+### Iron Man
+
+### Chuck Norris
+
+### Batman
+
+### Superman
+
+### Spiderman
+
+### Captain America
+
+### Arnold Schwarzenegger
+
+### James Bond
+
+### Diego Maradona
+
+### SectionTitleWithLongText
+
+### SectionTitleWithSimilarText
+
```

Comment:
The child section navigator component is provided by the base theme, not by the api docs theme so this test content should live in that smoke test website, too and not in the api docs smoke test.

Reply:
I moved it now into a new page in the smoke test under components called `Child section navigation`
90d6936